rails_programming/mailers_advanced_topics/advanced_topics.md

```diff
@@ -5,25 +5,12 @@ There are some topics that we just haven't had a chance to get into yet but will
 ### Learning Outcomes
 Look through these now and then use them to test yourself after doing the assignment:
 
-* When do you need to use a singular Resource vs a plural Resources in your router?
-* What is the "missing" route when using a singular Resource? (there are only 6 when you `$ rake routes`)  What else is missing from many of the other routes?
-* Why would you use nested routes?
-* What order do you specify their respective IDs? What are they called in `params`?
-* Why might you use a "member" route?
-* How are "member" and "collection" routes incredibly similar?  Slightly different?
-* How do you set up a redirect route that also passes along any parameters?
-* How do you name a route using an alias?
-* Why might you want to use nested or multiple layouts?
-* How would you (roughly) go about implementing this?
-* How can you pass variables between your layouts?
-* How do you `#yield` to `#content_for` content?
+* When do you use a singular Resource in your router and when do you use plural Resources?
+* When would you use nested routes?
+* When would you use a "member" route? A "collection" route?
+* When would you use multiple layouts?
```

Comment:
These still aren't *quite* lining up with our current Learning Outcome direction, I would check out [our wiki post about Learning Outcomes and Knowledge Checks](https://github.com/TheOdinProject/curriculum/wiki/Learning-Outcomes-vs-Knowledge-Checks) if you haven't yet.

If you already have but aren't sure what it is we're looking for (it's difficult to describe) I could provide some suggestions, but I don't want to take that agency away from you unless you request it 🙂

Reply:
So are you saying that those aren't appropriately high-level questions?

Reply:
Correct, that is what I'm saying. As an example of what are think are high level enough "What is metaprogramming?" and "What are Design Patterns?" seem great. If that still isn't clear though please let me know.